services:
  map_seg_name: make_offline_mask
  dl_seg_name: make_dl_mask

topics:
  input:
    gnss: "/anafi/gnss_location"
    height: "/anafi/height"
  output: 
    safe_points: "/segmentation/safe_points/camera_frame"

settings:
  interval_between_generation: 1.0
  enable_offline_map_segmentation: True
  enable_dl_segmentation: True
  min_altitude_to_use_dl_segmentation: 10
  min_safe_metric_dist: 5
  stride_in_search: 10
  min_altitude_to_generate_safe_points: 2

debug: True